IBM/ibm-common-service-operator
# add cert manager smoke test to setup singleton and preload data

For issue 59713, added a wait for cert manager operand webhook to come ready to setup singleton and then a smoke test where it tries to create a test issuer and certificate. This was added to the end of the script. Added just the smoke test to preload data's prereqs section so the script will error out if these items cannot be created

Testing setup singleton:
- run setup singleton to fresh install cert manager
- verify script completes and the success message "Cert manager is ready." appears at the end of the script

testing preload:
- before installing cert manager, run the script
- verify the script errors out during prereqs with the message: "Failed to create test issuer. Verify cert manager is installed and ready on the cluster then re-run the preload script."
- Install cert manager
- wait for it to come ready
- run preload again, verify the preload checks complete
- Note: you do not need to have any cs installed for the preload script to complete prereq checks. It will error out after the part we are looking at (which is expected).
